nxdialog/man/nxdialog.1

```diff
@@ -0,0 +1,72 @@
+'\" -*- coding: utf-8 -*-
+.if \n(.g .ds T< \\FC
+.if \n(.g .ds T> \\F[\n[.fam]]
+.de URL
+\\$2 \(la\\$1\(ra\\$3
+..
+.if \n(.g .mso www.tmac
+.TH nxdialog 1 "Jan 2019" "Version 3.5.99.18" "NX Dialog"
+.SH NAME
+nxdialog \- NX Dialog Helper
+.SH SYNOPSIS
+'nh
+.fi
+.ad l
+\fBnxdialog\fR \fI[-h] [--dialog DIALOG_TYPE] [--message TEXT]
+         [--caption CAPTION] [--display DISPLAY] [--parent AGENTPID]
+         [--window WINDOW] [--class DLGCLASS] [--local]
+         [--allowmultiple]\fR
+
+.SH DESCRIPTION
+\fBnxdialog\fR is a GUI helper that can be used with nxagent to provide
+user interaction on certain events inside nxagent. It adds graphical
+widgets (menus, dialog boxes, etc.) to nxagent that allow a smarter user
+interaction with the agent.
+.PP
+If your \fBnxagent\fR is not using \fBnxdialog\fR by default, then set
```

Comment:
Rephrase: nxagent is not using nxdialog by default, set NX_CLIENT to make the nxdialog path known to nxagent.

Reply:
> Rephrase: nxagent is not using nxdialog by default, set NX_CLIENT to make the nxdialog path known to nxagent.

got that